src/neural/network_blas.cc

```diff
@@ -61,7 +61,7 @@ class BlasComputation : public NetworkComputation {
       float value = plane.value;
       const uint64_t one = 1;
       for (int i = 0; i < 64; i++)
-        input_data_[index++] = ((plane.mask & (one << i)) == 0) ? 0 : value;
+        input_data_[index++] = plane.mask & (one << i) ? value : 0;
```

Comment:
Actually, MS visual studio requires == 0, else it's warning:
https://msdn.microsoft.com/en-us/library/b6801kcy.aspx

It's silly, I agree.
I think it's fine to keep it with == 0, as there are plenty of MSVC warnings in other places and noone cleaned them up yet.

Reply:
Rolled back. Using the !=0 which is as efficient on assembly level.